### DISCUSSION TITLE
Yi Outdoor Camera 1080p 9CUS

### DISCUSSION THREAD

**Discussion by STnmsk** [roleoroleo/yi-hack-Allwinner-v2#669] [Q&A]
Hello!
I have Yi Outdoor Camera 1080p, Serial number - 9CUS, Firmware - 3.0.0*
Can I install this hack? If not, did yor plan to add this camera in your hack?

**Comment by roleoroleo** [comment]
fw 3.0.0...
Probably this cam is an old platform. Not compatible with Allwinner.
Please check yi-hack-v5.
